recipes/outcome/all/conanfile.py

```diff
@@ -0,0 +1,31 @@
+import os
+from conans import ConanFile, tools
+
+class OutcomeConan(ConanFile):
+    name = "outcome"
+    homepage = "https://github.com/ned14/outcome"
+    description = "Provides very lightweight outcome<T> and result<T>"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("outcome", "result")
+    settings = "compiler"
+    no_copy_source = True
+
+    _source_subfolder = "source_subfolder"
+
+    def configure(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, "14")
```

Comment:
```suggestion
            tools.check_min_cppstd(self, "14")
        minimum_version = {
            "clang": "3.9",
            "gcc": "6",
            "Visual Studio": "15.0",
        }.get(str(self.settings.compiler))

        if not minimum_version:
            self.output.warn(
                "Unknown compiler {} {}. Assuming compiler supports C++14.".format(self.settings.compiler,                                                                                   self.settings.compiler.version))
        else:
            version = tools.Version(self.settings.compiler.version)
            if version < minimum_version:
                raise ConanInvalidConfiguration(
                    "The compiler {} {} does not support C++14.".format(self.settings.compiler,
                                                                       self.settings.compiler.version))
```

outcome requires GCC 6 at least.